cell_classification/segmentation_data_prep.py

```diff
@@ -209,18 +210,44 @@ def check_input(self):
         else:
             self.selected_markers = self.selected_markers
 
+            # check if selected markers are in conversion matrix
+            if not set(self.selected_markers).issubset(set(self.conversion_matrix.columns)):
```

Comment:
We created helper functions for checking the different ways certain arguments relate to others (must be a subset, must be equal, must be equal and in the same order, etc): https://github.com/angelolab/toffy/blob/main/toffy/rosetta.py#L83. This function gives some good examples of how to use them. In addition, because these are user-facing error checks, in addition to reporting the error, it also will find and print out the specific list items that are different. 

Right now `ark` has a lot of requirements, so long-term I don't think we want to import it. However, we're going to be moving all of these helper functions to their own repo next month, so for now I think it's okay.